plugins/modules/vmware_guest_instant_clone.py

```diff
@@ -0,0 +1,420 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# Copyright: (c) 2021, Anant Chopra <chopraan@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_guest_instant_clone
+short_description: Instant Clone VM
+
+description:
+- This module can be used for Creating a powered-on Instant Clone of a virtual machine.
+- All variables and VMware object names are case sensitive.
+
+options:
+  name:
+    description:
+      - Name of the Cloned virtual machine.
+    type: str
+    aliases: ['vm_name']
+    required: True
+  parent_vm:
+    description:
+      - Name of the parent virtual machine.
+      - This is a required parameter, if parameter C(uuid) or C(moid) is not supplied.
+    type: str
+  uuid:
+    description:
+      - UUID of the vm instance to clone from, this is VMware's unique identifier.
+      - This is a required parameter, if parameter C(parent_vm) or C(moid) is not supplied.
+    type: str
+  moid:
+    description:
+      - Managed Object ID of the vm instance to manage if known, this is a unique identifier only within a single vCenter instance.
+      - This is required if C(parent_vm) or C(uuid) is not supplied.
+    type: str
+  use_instance_uuid:
+    description:
+      - Whether to use the VMware instance UUID rather than the BIOS UUID.
+    default: false
+    type: bool
+  hostname:
```

Comment:
`hostname`, `username`, `password`, `port`, and `validate_certs` is not required when `doc_fragments` included.

Reply:
@Akasurde After removing these, I am getting the following errors in tests:
```
ERROR: Found 4 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/vmware_guest_instant_clone.py:0:0: doc-default-does-not-match-spec: Argument 'validate_certs' in argument_spec defines default as (False) but documentation defines default as (True)
ERROR: plugins/modules/vmware_guest_instant_clone.py:0:0: doc-required-mismatch: Argument 'hostname' in argument_spec is required, but is not documented as being required
ERROR: plugins/modules/vmware_guest_instant_clone.py:0:0: doc-required-mismatch: Argument 'password' in argument_spec is required, but is not documented as being required
ERROR: plugins/modules/vmware_guest_instant_clone.py:0:0: doc-required-mismatch: Argument 'username' in argument_spec is required, but is not documented as being required
```